README.md

```diff
@@ -9,3 +9,86 @@ $ bower install gaia-components/gaia-list
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-list/)
+
+## Usage
+
+Settings style list
+
+```
+<gaia-list>
+  <a>
```

Comment:
To imply proper semantics, every row should be wrap in `<li>`, we could improve that in another bug